.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports =  {
+    semi:  true,
+    trailingComma:  'all',
```

Comment:
we might want to disable trailing find at least for json files as that is invalid and will cause errors due to failures to load, e.g. package. Json

Reply:
It does not seem to add trailingComma into *.json files by default